db/seeds.rb

```diff
@@ -105,183 +105,43 @@ def get_rating
 end
 
 Exercise.create.tap do |e|
+  e.id = 1
   e.points = 15
   e.exercise_category_id = ExerciseCategory.find_by_name("Introduce Explaining Variable").id
-  e.code_to_refactor = <<CODE
-def price
-  # price is base price - quantity discount + shipping
-  return @quantity * @item_price -
-  [0, @quantity - 500].max * @item_price * 0.05 +
-  [@quantity * @item_price * 0.1, 100.0].min
-end
-CODE
-  e.solution = <<CODE
-def price
-  base_price = @quantity * @item_price
-  quantity_discount = [0, @quantity - 500].max * @item_price * 0.05
-  shipping = [base_price * 0.1, 100.0].min
-  return base_price - quantity_discount + shipping
-end
-CODE
```

Comment:
Yeah, I agree. I think its fine, it is just necessary to add an id to the factory and clarify that it is creating an exercise of a concrete category (check factories/exercises.rb)